plugins/destination/mssql/client/write.go

```diff
@@ -0,0 +1,118 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/cloudquery/plugin-sdk/plugins/destination"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/specs"
+)
+
+func (c *Client) Write(ctx context.Context, tables schema.Tables, resources <-chan *destination.ClientResource) error {
+	query := upsert
+	if c.spec.WriteMode == specs.WriteModeAppend {
+		query = insert
+	}
+
+	batchSize := c.spec.BatchSize
+
+	batch := make([]*stmt, 0, batchSize)
+	for r := range resources {
+		table := tables.Get(r.TableName)
+		if table == nil {
+			panic(fmt.Errorf("table %s not found", r.TableName))
```

Comment:
I don't think you need to panic here. just return an error it will stop the sync

Reply:
I copy-pasted it from [postgres](https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/postgresql/client/write.go#L64), I can change it to just returning the err.